Gems/Multiplayer/Code/Source/Components/NetworkRigidBodyComponent.cpp

```diff
@@ -143,8 +143,16 @@ namespace Multiplayer
 
     void NetworkRigidBodyComponentController::OnTransformUpdate()
     {
-        AzPhysics::RigidBody* rigidBody = GetParent().m_physicsRigidBodyComponent->GetRigidBody();
-        SetLinearVelocity(rigidBody->GetLinearVelocity());
-        SetAngularVelocity(rigidBody->GetAngularVelocity());
+        // Even though this component requires a rigid body, OnTransformUpdate can get called while the entity is being destroyed and may already have its rigidbody removed
```

Comment:
That is because the event handler isn't disconnected in NetworkRigidBodyComponentController::OnDeactivate?

Reply:
Disconnecting would be good; added! Unfortunately, this crash will still occur, TransformComponent updates its transform on level unload because the parent is deactivated, that seems to happens before the entity component's themselves call Deactivate(), yet after the PhysicSceneInterface tears down all it's rigid bodies, ooph!

Reply:
Update: The null-check for rigidbody already exist in dev. Merged latest.